packages/itwin/grouping-mapping-widget/src/widget/components/utils.tsx

```diff
@@ -152,3 +153,15 @@ export const LoadingSpinner = () => {
     </div>
   );
 };
+
+const prefixUrl = (baseUrl?: string, prefix?: string) => {
+  if (prefix && baseUrl) {
+    return baseUrl.replace("api.bentley.com", `${prefix}api.bentley.com`);
```

Comment:
`${prefix}-api.bentley.com` missing the dash after prefix.

Reply:
This will be fine. In prod it's just API.bentley.com not prod-api.bentley.com

Reply:
Nvm I guess it depends one what you guys use as prefix. Everywhere else we include the dash in it

Reply:
I prefer the whole URL to future proof. Either we get rid of imodel-browser-react or add support for custom URL's. As of now it only lets you change the prefix and they also don't include the dash.

Reply:
Added a dash.